src/Symfony/Component/Serializer/Normalizer/DateTimeZoneNormalizer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes an object {@see \DateTimeZone} to a timezone string.
+ *
+ * @author Jérôme Desjardins <jewome62@gmail.com>
+ */
+class DateTimeZoneNormalizer implements NormalizerInterface, DenormalizerInterface, CacheableSupportsMethodInterface
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException
+     */
+    public function normalize($object, $format = null, array $context = [])
+    {
+        if (!$object instanceof \DateTimeZone) {
+            throw new InvalidArgumentException('The object must implement the "\DateTimeZone".');
```

Comment:
The object must be an instance of "\DateTimeZone".

Reply:
(Why this exception if `supportsNormalization()` guarantees that `$object` is an instance of DateTimeZone ?)

Reply:
I have reproduce the same test from DateTimeNormalizer and DateIntervalNormalizer
I think that's implemented if normalize is call without call support before